Makefile

```diff
@@ -48,8 +48,8 @@ image-aro-multistage:
 	docker build --no-cache -f Dockerfile.aro-multistage -t $(ARO_IMAGE) .
 
 image-fluentbit:
-	docker build --no-cache --build-arg VERSION=1.6.10-1 \
-	  -f Dockerfile.fluentbit -t ${RP_IMAGE_ACR}.azurecr.io/fluentbit:1.6.10-1 .
+	docker build --no-cache --build-arg VERSION=1.7.8-1 \
+	  -f Dockerfile.fluentbit -t ${RP_IMAGE_ACR}.azurecr.io/fluentbit:1.7.8-1 .
```

Comment:
Do we need to change the publish-image-fluentbit version down below? https://github.com/Azure/ARO-RP/pull/1551/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L73

Reply:
Yup, thanks.  Moved to a var so we can set it in one place.